applications/serial_lte_modem/modem-trace-uart.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Enable uart1 for tracing. */
+&uart1 {
+	status = "okay";
+	current-speed = < 1000000 >;
```

Comment:
Should this have hw-flow-control; as well? RTT-logging backend is used by default and that does not work with this without hw-flow-control.